docs/001-devices.apib

```diff
@@ -56,8 +59,37 @@ should be included in the main data structure alongside `hostname` and `plan`.
 ## Device [/devices/{id}]
 A single device object.
 
+    + Attributes (object)
+        + hostname (string)
+        + plan (enum[string])
+            + Members
+                + baremetal_0 - Type 0
+                + baremetal_1 - Type 1
+                + baremetal_3 - Type 3
+        + billing_cycle (enum[string])
+            + Members
+                + hourly
+        + facility (enum[string]) - Facility Location
+            + Members
+                + ewr1 - Parsippany, NJ
+        + operating_system (enum[string]) - Operating system to install. Operating systems availables may vary depending of the selected plan.
```

Comment:
Should perhaps be something like:
> Operating system to install. Valid operating systems vary depending on the selected plan.

And is this part a typo? The `c` part in particular. Regardless, I think we can just remove this part.
> The endpoint `c` provides the list of all operating systems, each of them has an attribute `provisionable_on` to know in which plans it is available.